src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -45,6 +53,25 @@ public DynamicallyAccessedMemberTypes GetParameterAnnotation (MethodDefinition m
 			return DynamicallyAccessedMemberTypes.None;
 		}
 
+		public DynamicallyAccessedMemberTypes GetMemberTypesForDynamicallyAccessedMemberAttribute (CustomAttributeSource source, ICustomAttributeProvider provider)
```

Comment:
There's no need to pass the `source` as a parameter - it's available as a field on "this".